### PR TITLE
add cat training

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: resources.yaml [hunk 1]
```diff
@@ -297,7 +297,12 @@ deployment:
     group: training-gie-0324
     image: htcondor-secondary
     secondary_htcondor_cluster: true
-
+ training-geno:
```
Comment (mira-miracoli) [inline resources.yaml:300]:
```suggestion
   training-geno:
```

File: resources.yaml [hunk 2]
```diff
@@ -297,7 +297,12 @@ deployment:
...
+      count: 2
+      flavor: c1.c28m225d50
+      start: 2024-05-07
+      end: 2024-05-10
+      group: training-genome-assembly-2024
```
Comment (mira-miracoli) [inline resources.yaml:305]:
```suggestion
      group: training-genome-assembly-2024
      image: htcondor-secondary
      secondary_htcondor_cluster: true
```

File: resources.yaml [hunk 3]
```diff
@@ -297,7 +297,14 @@ deployment:
...
     secondary_htcondor_cluster: true
-
+  training-geno:
+      count: 2
+      flavor: c1.c28m225d50
+      start: 2024-05-07
+      end: 2024-05-10
+      group: training-genome-assembly-2024
+      image: htcondor-secondary
+      secondary_htcondor_cluster: true
```
Comment (mira-miracoli) [inline resources.yaml:307]:
```suggestion
    image: htcondor-secondary
    secondary_htcondor_cluster: true
```

File: resources.yaml [hunk 4]
```diff
@@ -297,7 +297,14 @@ deployment:
...
     secondary_htcondor_cluster: true
-
+  training-geno:
+    count: 2
+    flavor: c1.c28m225d50
+    start: 2024-05-07
+    end: 2024-05-10
+    group: training-genome-assembly-2024
+    image: htcondor-secondary
+    secondary_htcondor_cluster: true
```
Comment (bgruening) [inline resources.yaml:307]: is that still needed?

Comment (mira-miracoli) [reply resources.yaml:307]: yes until we also get rid of this in vgcn infrastructure

Comment (mira-miracoli) [reply resources.yaml:307]: yes until we remove it in this repo

**Pull request merged**